debian/one-compiler.links

```diff
@@ -0,0 +1,26 @@
+# bin
+usr/share/one/bin/circle2circle usr/bin/circle2circle
+usr/share/one/bin/circle_partitioner usr/bin/circle_partitioner
+usr/share/one/bin/circle-quantizer usr/bin/circle-quantizer
+usr/share/one/bin/generate_bcq_metadata usr/bin/generate_bcq_metadata
+usr/share/one/bin/generate_bcq_metadata.py usr/bin/generate_bcq_metadata.py
```

Comment:
Is this right approach to put the python script in `/usr/bin`?
I don't know ONE compiler well,,, however I don't think the python scripts and cfg file should be here.